.travis.yml

```diff
@@ -30,7 +30,11 @@ script:
 - docker build -t e2e_tests -f docker/Dockerfile.mule .
 - docker run --rm -t e2e_tests mule/e2e.sh
 
+after_success:
+  - scripts/upload_coverage.sh || true
```

Comment:
My biggest question was going to be "what happens when their cloud chokes?" and we just shrug and ignore it. Sounds good to me.